src/components/help-icon/help-icon.md

```diff
@@ -0,0 +1,35 @@
+HelpIcon is a simple prebaked SVG and tooltip, as it is the standard way to demonstrate help text for components in Mavenlink.
+
+#### Basic Usage
+
+```jsx
+import HelpIcon from "./help-icon.jsx";
+
+<HelpIcon 
+  text="This is the text that appears from the icon" 
+  label="title text for the SVG" 
+  id="example-tooltip"
+/>
+```
+
+#### In-depth example
+
+More realistically, the `HelpIcon` is going to be used in conjunction with other elements, like inputs or paragraphs.
+
+Much like with `Tooltip`, please remember to apply `aria-describedby` on the component the help text is related to.
```

Comment:
I wonder if it's worthwhile to add a `useLayoutEffect` that effectively looks for the `aria-describedby` element. 🤔  Maybe a future iteration.

Reply:
What would the intent of it to actually be? Just scan the dom to make sure the element is there and raise an error if not or something?

Reply:
Yeah, exactly. It's basically a run-time linter so the dev is setting things up correctly. 

Pros:
- Our stuff is accessible by a forcing linter

Cons:
- CPU cycles wasted for checking validity of app on customer's computers

Reply:
I think the pros would outweigh the cons.. 

We would only have to do it once per instance of the tooltip, I imagine we would only do it on mount instead of on every render cycle, but there's definitely still a cost there.

I've got some refactors I want to make to the tooltip anyway (mostly investigating react portal), so I'll see if I can add this in that PR. Thanks for the feedback!